etc/workflows/duplicate-event.xml

```diff
@@ -12,18 +12,48 @@
     <![CDATA[
       <div id="workflow-configuration">
         <fieldset>
-          <input
-            id="numberOfEvents"
-            name="numberOfEvents"
-            type="number"
-            class="configField"
-            onkeypress="return event.charCode > 47"
-            oninput="checkValueInBounds()"
-            min="1"
-            value="1"
-            max="25"
+          <br/>
+          <div>
+            <label for="mpTitle" style="width: 150px;" >Title of Duplicate:</label>
+            <input id="mpTitle" name="mpTitle" type="text" style="width: 400px;" value=" " class="configField" />
+          </div>
+          <br/>
+          <div>
+            <label for="seriesId" style="width: 150px;" >Series Id:</label>
+            <input id="seriesId" name="seriesId" type="text" maxlength="255" style="width: 400px;" value=" " class="configField" />
+          </div>
+          <br/>
+          <div>
+            <label for="startDate" style="width: 150px;">StartDate (YYYY-MM-DD):</label>
+            <input id="startDate" name="startDate" type="text" style="width: 200px;" value=" " class="configField" />
+          </div>
+          <div>
+            <label for="startTime" style="width: 150px;">startTime (HH:MM:SS):</label>
+            <input id="startTime" name="startTime" type="text" style="width: 200px;" value=" " class="configField" />
```

Comment:
…and [`type=time`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/time)

Reply:
Same as above answer

Reply:
fixed with code revamped.